lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -65,8 +65,9 @@
     gluster:
         description:
             - "If I(True), hosts in this cluster will be used as Gluster Storage
-               server nodes, and not for running virtual machines."
-            - "By default the cluster is created for virtual machine hosts."
+               server nodes."
+            - "By default the cluster is created for virtual machine hosts only."
+            - "Note that you cannot add oVirt Host (RHVH) to a Gluster-enabled cluster"
```

Comment:
I think upstream it's called `oVirt node`,  downstream RHV-H

Reply:
I just copied the oVirt documentation from here, https://www.ovirt.org/documentation/admin-guide/chap-Clusters/ but you're probably right, I 'll repost.